CHANGELOG.md

```diff
@@ -773,3 +773,5 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#335](https://github.com/nix-rust/nix/pull/335))
 
 ## [0.5.0] 2016-03-01
+- Added `posix_fallocate`.
```

Comment:
This CHANGELOG entry is seriously dislocated.  It should be up in the "Unreleased" section.

Reply:
Oh, sorry